### PR TITLE
fix: update peer dependency range

### PR DESCRIPTION
**Pull request by rajtslegr**
issue #93

### CODE REVIEW

File: packages/msw-addon/package.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
     "typescript": "^4.5.2"
   },
   "peerDependencies": {
-    "msw": ">=0.35.0 <1.0.0"
+    "msw": ">=0.35.0 <1.1.0"
```
Comment (bmvantunes) [inline packages/msw-addon/package.json:43]: @rajtslegr taking into account that `msw` follows [semver](https://semver.org/) we can safely make the peerDependencies as `<2.0.0`

`msw` semver statement: https://github.com/mswjs/msw/blob/main/CHANGELOG.md

Comment (rajtslegr) [reply packages/msw-addon/package.json]: Thanks for suggestion! Done!

**Comment by DominicGBauer** [comment]
@bmvantunes can we please get this merged?

**Comment by bmvantunes** [comment]
@DominicGBauer I don't work on this project and I don't have merging rights

**Comment by philwolstenholme** [comment]
This change works for me when applied via patch-package. Without it the 'conflicting peer dependency' npm error makes things difficult unless you use the `--force` flag for your npm commands.

LGTM? @yannbf

**Comment by YuiSakamoto** [comment]
@yannbf 
It looks fine to me.
When will this pull request be merged and released?

**Review by kettanaito — APPROVED** [review]

**Comment by kettanaito** [comment]
Thanks for fixing this, @rajtslegr! 🎉 

We should also bump this to `> 1.0.0 < 2.0.0` sometime later to encourage people to stay up-to-date with MSW.

**Pull request merged**